package/src/components/MessageInput/MessageInput.tsx

```diff
@@ -51,6 +59,7 @@ import {
 import { isImageMediaLibraryAvailable, triggerHaptic } from '../../native';
 import type { Asset, DefaultStreamChatGenerics } from '../../types/types';
 import { AutoCompleteInput } from '../AutoCompleteInput/AutoCompleteInput';
+import { CreatePollContent } from '../Poll/CreatePollContent';
```

Comment:
This isn't customizable using a UI prop?

Reply:
Now is !